### PR TITLE
bugfix: fix prometheus server post process callback

### PR DESCRIPTION
**Pull request by edocevol**
…n't by `log` method.**Please provide issue(s) of this PR:**
Fixes #446 

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration
- [ ] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [ ] Naming
- [ ] HealthCheck
- [x] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by edocevol** [comment]
For #446

**Review by chuntaojun — APPROVED** [review]

File: plugin/discoverstat/discoverlocal/discovercall.go [hunk 1]
```diff
@@ -76,6 +76,4 @@ func (d *DiscoverCallStatis) log() {
 	}
 
 	d.logger.Info(buffer.String())
-
-	d.statis = make(map[Service]time.Time)
```
Comment (chuntaojun) [inline plugin/discoverstat/discoverlocal/discovercall.go:80]: why remove it ？

Comment (chuntaojun) [reply plugin/discoverstat/discoverlocal/discovercall.go:80]: map只是暂存这个周期的服务调用数据，log输出之后，在开启下一轮次的数据打印暂存，移除之后反而会导致这个map可能无限增大

Comment (chuntaojun) [reply plugin/discoverstat/discoverlocal/discovercall.go:80]: you need to rollback this modify

Comment (edocevol) [reply plugin/discoverstat/discoverlocal/discovercall.go:80]: got

**Pull request merged**